src/forder.c

```diff
@@ -39,7 +39,7 @@ static int gs_n = 0;                // the number of groups found so far (how mu
 static int **gs_thread=NULL;        // each thread has a private buffer which gets flushed to the final gs appropriately
 static int *gs_thread_alloc=NULL;
 static int *gs_thread_n=NULL;
-static int *global_TMP=NULL;               // UINT16_MAX*sizeof(int) for each thread; used by counting sort in radix_r()
```

Comment:
I renamed the variable, because it made it easier to distinguish from the other block-local `TMP`(s?) during debugging. But yes, for consistency I probably should have named it back, after I was done.